extensions/arc/deployment/src/test/java/io/quarkus/arc/test/unproxyable/RequestScopedFinalMethodsTest.java

```diff
@@ -26,7 +26,8 @@ public class RequestScopedFinalMethodsTest {
     @RegisterExtension
     public static QuarkusUnitTest container = new QuarkusUnitTest()
             .withApplicationRoot((jar) -> jar
-                    .addClasses(RequestScopedBean.class));
+                    .addClasses(RequestScopedBean.class, OtherRequestScopedBean.class, OtherRequestScopeBeanProducer.class,
+                            StringProducer.class, StringObserver.class));
```

Comment:
Why do you add the `StringProducer.class` and `StringObserver.class` to the test? Is it used somewhere?

Reply:
Good point, they do seem useless... removing